app/models/organization.rb

```diff
@@ -9,28 +10,35 @@ def initialize(init_params)
     @name = init_params['name']
     @description = init_params['description']
     @imageUrl = init_params['profileImageUrl']
-    @categories = init_params['type']
-    @interests = init_params['categories']
+    @categories = init_params['categories']
+    @interests = init_params['interests']
   end
 
   def self.get_organizations(num, categories, interests)
     orgs = self.get_orgs_from_api()
-
+    csv_text = File.read('lib/interest.csv')
+    interest = CSV.parse(csv_text, headers: true)
     top_orgs = []
     orgs.each do |i|
       org_categories = i['categories']

```

Comment:
When I stuck a binding.pry I found that 
``` 
org_categories => [{"categoryId"=>2693, "categoryName"=>"y ASUC Student Activity Groups"},
 {"categoryId"=>5119, "categoryName"=>"Spiritual Student Organizations"},
 {"categoryId"=>7850, "categoryName"=>"Undergraduate Organizations"}]
```
I think we only ever need to match for the Spiritual Student Organizations chunk for each org. There might be a way to not loop over org_categories and instead just grab the category we need